_includes/pbc/all/install-features/202304.0/install-the-spryker-core-back-office-warehouse-user-management-feature.md

```diff
@@ -0,0 +1,98 @@
+This document describes how to integrate the Spryker Core Back Office + Warehouse User Management Feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Picking App feature API.
+
+## Prerequisites
+
+To start feature integration, integrate the required features and Glue APIs:
+
+| NAME                      | VERSION          |
+|---------------------------|------------------|
+| Spryker Core API          | {{site.version}} |
```

Comment:
Please check other IGs and confirm that we really need `Spryker Core API` dep.
As for me, it's already dep to `Spryker Core Back Office` and `Warehouse User Management` features and we can skip it.